src/main/java/GeometricObject.java

```diff
@@ -0,0 +1,20 @@
+public abstract class GeometricObject {
+
+    private String color;
+    private int area;
```

Comment:
that should be calculated inside of method getArea()

Reply:
done